apis/core/types_execution.go

```diff
@@ -86,10 +86,6 @@ type ExecutionStatus struct {
 	// +optional
 	DeployItemCache *DeployItemCache `json:"deployItemCache,omitempty"`
 
-	// ExecutionGenerations stores which generation the execution had when it last applied a specific deployitem.
-	// So in this case, the observedGeneration refers to the executions generation.
-	ExecutionGenerations []ExecutionGeneration `json:"execGenerations,omitempty"`
```

Comment:
If we remove this field, do we still need the `ExecutionGeneration` structure below?

Reply:
I have removed this